.github/workflows/build.yml

```
@@ -20,6 +20,9 @@ jobs:
         script: [ 'build-base-k8s-jenkins-agent.sh', 'build-base-k8s-jenkins-coverage-agent.sh', 'build-base-swarm-jenkins-agent.sh', 'build-base-swarm-jenkins-coverage-agent.sh', 'build-edt-swarm-agent.sh', 'build-edt-k8s-agent.sh', 'build-oscript-k8s-agent.sh', 'build-oscript-swarm-agent.sh', 'build-server.sh', 'build-executor.sh']
 
     steps:
+    - name: Maximize build space
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)